unity/Assets/Scripts/AgentManager.cs

```diff
@@ -316,24 +321,15 @@ public BaseFPSAgentController ReturnPrimaryAgent()
 		gameObject.transform.eulerAngles = action.rotation;
 		gameObject.transform.position = action.position;
 
-        float fov;
+		// default to 90 fov on third party camera if value is too small or large
+		camera.fieldOfView = ClampFieldOfView(action.fieldOfView);
 
-        if(action.fieldOfView <= 0 || action.fieldOfView > 180)
-        {
-            //default to 90 fov on third party camera if nothing passed in, or if value is too large
-            fov = 90f;
-        }
-        else
-        {
-            fov = action.fieldOfView;
-        }
         if (action.orthographic) {
+			// REVIEW: should the orthographicSize use the clamped fov or the original value passed in?
```

Comment:
REVIEW: should the orthographicSize use the clamped fov or the original value passed in?

Reply:
Orthographic should just use the FoV as passed by the action

Reply:
added a note